### PR TITLE
Add initial code from YW :bat:

### PR DESCRIPTION
**Pull request by niroz89**
Comes with 🦇 for chart testing.

### CODE REVIEW

File: stable/consul-esm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Default values for consul-esm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
+
+image:
+  repository: basisai/consul-esm
```
Comment (niroz89) [inline stable/consul-esm/values.yaml]: @lawliet89 this one should be `hashicorp/consul-esm` ?

File: stable/consul-esm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,109 @@
+# Default values for consul-esm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
```
Comment (niroz89) [inline stable/consul-esm/values.yaml]: shall I change to 2 by default ?

File: stable/consul-esm/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+appVersion: "0.4.0"
+description: Consul ESM for external service health checks in Consul
+name: consul-esm
+version: 0.3.1
+home: https://github.com/hashicorp/consul-esm
```
Comment (niroz89) [inline stable/consul-esm/Chart.yaml]: any suggestion on `appVersion` and `version` ? `appVersion` we use the consul-esm docker tag ? and for version separate semver ?

File: stable/consul-esm/values.yaml (hunk 1 above)
Comment (lawliet89) [reply stable/consul-esm/values.yaml]: They don't have a chart 😓

File: stable/consul-esm/values.yaml (hunk 2 above)
Comment (lawliet89) [reply stable/consul-esm/values.yaml]: Sure

File: stable/consul-esm/Chart.yaml (hunk 3 above)
Comment (lawliet89) [reply stable/consul-esm/Chart.yaml]: Yes

File: stable/consul-esm/values.yaml (hunk 1 above)
Comment (niroz89) [reply stable/consul-esm/values.yaml]: above is representing the docker image, which is in https://hub.docker.com/r/hashicorp/consul-esm

Comment (lawliet89) [reply stable/consul-esm/values.yaml]: Oh, sorry. Can you check if I made some special changes? Otherwise let's point back to the official one.

Comment (niroz89) [reply stable/consul-esm/values.yaml]: https://github.com/SPHTech-Platform/terraform-kubernetes-consul/blob/8dbf84be0733f605eb6ba2e6f29f9e8e57b53f4d/modules/esm/variables.tf#L85 -> here you are using `hashicorp/consul-esm` , so I assume nothing special there.

**Review by lawliet89 — APPROVED** [review]

**Pull request merged**